scripts/tests/api-docs.js

```diff
@@ -0,0 +1,177 @@
+const chai = require('chai')
+const glob = require("glob");
+const fs = require("fs");
+const path = require("path");
+const cheerio = require('cheerio');
+const htmlparser2 = require('htmlparser2');
+const process = require('process')
+
+const expect = chai.expect
+
+const pkg = process.argv[5].replace("--pkg=", "");
+
+console.log("pkg", pkg);
+
+if (!pkg) {
+  console.error("package not set", 1);
+}
+
+const modules = getModules(pkg);
+modules.forEach(mod => {
+  const paths = glob.sync(`./public/registry/packages/${pkg}/api-docs/${mod}/**/*.html`, {
+    ignore: [
+      `./public/registry/packages/${pkg}/api-docs/${mod}/index.html`
+    ],
+  });
+  paths.forEach(p => {
+    const fileContent = fs.readFileSync(p, "utf-8").toString();
+    const dom = htmlparser2.parseDocument(fileContent);
+    const $ = cheerio.load(dom);
+
+    // Ignore funtion pages for now.
+    if (!path.basename(path.dirname(p)).startsWith("get")) {
+
+      describe(constructPageRoute(p), function () {
+        // Verify page has a title and it is an h1 that contains the package and
+        // module name.
+        describe("h1 title", function () {
+          const h1s = $("h1");
+
+          it("contains exactly 1 h1", function () {
+            expect(h1s.length).to.equal(1);
+          })
+          it("contains the package name", function () {
+            expect(h1s.text().toLowerCase()).to.have.string(pkg);
+          })
+          it("contains the module name", function () {
+            expect(h1s.text().toLowerCase()).to.have.string(mod);
+          })
+        })
+
+        // Verify sections exist in correct order
+        describe("Sections", function () {
+          const headings = [];
+          $("section.docs-content > h2").each((i, elm) => {
```

Comment:
agreed. I updated this save the `h` and `ph` since `ph` is referring to possible headings that could exist on the page and `h` refers to the headings that are actually there.